squid/values.yaml

```diff
@@ -55,6 +55,39 @@ service:
   # This sets the ports more information can be found here: https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports
   port: 3128
 
+# Squid Prometheus Exporter Configuration
+# This enables monitoring of Squid metrics via Prometheus
+squidExporter:
+  # Enable or disable the squid-exporter sidecar
+  enabled: true
+  # Squid exporter container image
+  image:
+    repository: boynux/squid-exporter
+    tag: "v1.13.0"
+    pullPolicy: IfNotPresent
+  # Port on which the exporter will serve metrics
+  port: 9301
+  # Metrics path for Prometheus scraping
+  metricsPath: "/metrics"
+  # Squid connection settings for the exporter
+  squidHostname: "localhost"
+  squidPort: 3128
```

Comment:
We need to make sure values in this file cannot contardict or duplicate one another - sicne we have another value here already configuring the squid port, we shouldn't have another configuration for it here. 

Same goes for the host name, since we're running in a POD, its alwas going to be `localhost` so no point in having a value here that can just break the monitoring for us.